packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -202,8 +210,12 @@ async function main(): Promise<number | void> {
     jestCli = jestCli.withArgs(['--testPathIgnorePatterns', 'relationMode-in-separate-gh-action'])
   }
 
+  if (process.env.PRISMA_CLIENT_ENGINE_TYPE && !args['--engine-type']) {
+    throw new Error('Functional tests expect --engine-type to be explicitly set, not via env var')
+  }
+
   try {
-    if (args['-u']) {
+    if (args['--updateSnapshot']) {
```

Comment:
`-u` did not work anymore, maybe because of a prev. PR. Fixed it.

Reply:
Most likely my PR https://github.com/prisma/prisma/pull/21794, thanks!